docs/collecting-data/collecting-from-own-applications/python-tracker/tracking-specific-events/index.md

```diff
@@ -1,528 +1,282 @@
-| `value` | A value to provide numerical data about the event | No | Int or Float |
-| `context` | Custom context for the event | No | List(SelfDescribingJson) |
-| `tstamp` | When the structured event occurred | No | Positive integer |
-| `event_subject` (since v0.9.0) | The subject for the event | No | Subject |
-
-Example:
+event_subject = Subject().set_user_id("1234")
+page_view = PageView(
+  page_url="https://www.snowplow.io",
+  page_title="Homepage",
+  event_subject=event_subject
+)
 
-```python
-t.track_struct_event("shop", "add-to-basket", None, "pcs", 2)
+t.track(page_view)
 ```
 
-### Track link clicks
+### Custom context
 
-Use `track_link_click()` to track individual link click events. Arguments are:
-
-| **Argument** | **Description** | **Required?** | ******Type****** |
-| --- | --- | --- | --- |
-| `target_url` | The URL of the page | Yes | Non-empty string |
-| `element_id` | ID attribute of the HTML element | No | String |
-| `element_classes` | Classes of the HTML element | No | List(string) |
-| `element_target` | Target element | No | String |
-| `element_content` | The content of the HTML element | No | String |
-| `context` | Custom context for the event | No | List(SelfDescribingJson) |
-| `tstamp` | When the pageview occurred | No | Positive integer |
-| `event_subject` (since v0.9.0) | The subject for the event | No | Subject |
+Custom context can be used to augment any standard Snowplow event type, including self describing events, with additional data.
 
-Basic example:
+Custom context can be added as an extra argument to any of Snowplow’s Event classes.
 
-```python
-t.track_link_click("http://my-target-url2/path")
-```
+When you track an event, some of the data that you track will be specific to that event. A lot of the data you want to record, however, will describe entities that are tracked across multiple events. For example, a media company might want to track the following events:
 
-Advanced example:
+- User views video listing
+- User plays video
+- User pauses video
+- User shares video
+- User favorites video
+- User reviews video
 
-```python
-t.track_link_click("http://my-target-url2/path", "element id 2", None, "element target", "element content")
-```
+Whilst each of those events is a different type, all of them involve capturing data about the user and the video. Both the 'user' and 'video' are entities that are tracked across multiple event types. Both are candidates to be "custom context". You as a Snowplow user can define your own custom contexts (including associated schemas) and then send data for as many custom contexts as you wish with _any_ Snowplow event. So if you want, you can define your own "user context", and then send additional user data in that object with any event. Other examples of context include:
 
-### Track add-to-cart events
+- articles
+- videos
+- products
+- categories
+- pages / page_types
+- environments
```

Comment:
Again this all feels like it's already explained elsewhere in the docs, link to the docs for contexts and keep this page just for what's specific to the python tracker?